ci/prow/config.yaml

```diff
@@ -5630,6 +5630,54 @@ periodics:
     - name: housekeeping-github-token
       secret:
         secretName: housekeeping-github-token
+- cron: "30 07 * * *"
+  name: ci-knative-serving-recreate-clusters
+  agent: kubernetes
+  decorate: true
+  extra_refs:
+  - org: knative
+    repo: test-infra
+  spec:
+    containers:
+    - image: gcr.io/knative-performance/update-serving:v20190809-c290f606fc20
+      imagePullPolicy: Always
+      command:
+      - "/workspace/tools/recreate-serving/recreate.sh"
```

Comment:
we should not be using this image. We can directly run the scripts in serving like we do with perf-tests.sh https://github.com/knative/serving/blob/master/test/performance/tools/update-serving/update.sh

Reply:
Done.